src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -100,11 +120,17 @@ const LostAndFoundCard = () => {
   );
 
   // Component defining each row of the report grid
-  const reportRow = (report: MissingItemReport) => (
+  const reportRow = (report: MissingItemReport, isFoundSection: boolean = false) => (
     <Card
-      className={`${styles.dataRow} ${
-        report.status.toLowerCase() === 'active' ? styles.clickableRow : ''
+      className={`${isFoundSection && report.status.toLowerCase() === 'found' ? styles.dataFoundRow : styles.dataRow} ${
+        styles.clickableRow
       }`}
+      onClick={() => {
+        // Navigate to the edit page for found items
+        if (isFoundSection && report.status.toLowerCase() === 'found') {
+          navigate(`/lostandfound/${report.recordID}`);
```

Comment:
I think it makes sense for just the found ones, upon speaking with Erik we thought it would be better that way because, we dont want the person to click on the item, go to the main page, before clicking again to go to click on pickup, it makes sense if it dircetly goes there for a found item. Since more times than not, only 1 item would be found per person

Reply:
Valid, yeah I agree 👍